app/src/main/java/com/github/houseorganizer/houseorganizer/calendar/CalendarAdapter.java

```diff
@@ -58,11 +59,15 @@ public void showAddEventDialog(Context ctx, DocumentReference currentHouse, Stri
                 .setTitle(R.string.event_creation_title)
                 .setView(dialogView)
                 .setPositiveButton(R.string.add, (dialog, id) -> {
+                    EspressoIdlingResource.increment();
```

Comment:
thats a really smart way to get around things, im assuming it only does something when tested since its espresso so thats amazing

Reply:
Good call! Done in last commit